gradio/components.py

```diff
@@ -480,6 +480,7 @@ class Number(Changeable, Submittable, IOComponent):
     Creates a numeric field for user to enter numbers as input or display numeric output.
     Preprocessing: passes field value as a {float} or {int} into the function, depending on `precision`.
     Postprocessing: expects an {int} or {float} returned from the function and sets field value to it.
+    Examples: Float or int representing value.
```

Comment:
I called it `Examples` to go with the pattern of one word tags. Open to other suggestions.

Reply:
Yeah agreed, I would prefer "Example format" instead of "Examples"

Reply:
Sounds good! I'm going to go with `Examples-format` since right now the documentation-building code assumes the tag is only one word long.